nnpackage/spec/10_packaging_and_manifest.md

```diff
@@ -79,10 +81,11 @@ The first element from the array will be the default model to be executed.
 
 It can have the values (case-sensitive) in following table.
 
-| name   | description            |
-|--------|------------------------|
-| tflite | tensorflow lite schema |
-| circle | nnpackage schema       |
+| name   | description              |
+|--------|--------------------------|
+| tflite | tensorflow lite schema   |
+| circle | nnpackage schema         |
+| bin    | backend dependent binary |
```

Comment:
Maybe it is not yet for review. However, please let me leave comment since I will forget to do later. : )
`tflite` and `circle` are also binary in term that it is not human readable. Thus, I prefer to use more specific names. Maybe `prebuilt` (as you wrote in title) or `backend-dependent` (as you described in table) would be better.